src/lib/snark_pool/dune

```diff
@@ -3,7 +3,7 @@
  (public_name snark_pool)
  (inline_tests)
  (library_flags -linkall)
- (libraries core extlib protocols async coda_base dyn_array)
+ (libraries core extlib protocols async coda_base dyn_array transaction_snark)
```

Comment:
Maybe we don't need this now that we removed the constraint from snark_pool.ml

Reply:
removed this library, pushed code